catroid/src/main/java/org/catrobat/catroid/common/defaultprojectcreators/ArDroneProjectCreator.java

```diff
@@ -79,25 +89,17 @@ public Project createDefaultProject(String projectName, Context context, boolean
 		Script whenSpriteTappedScript = new WhenScript();
 		whenSpriteTappedScript.addBrick(brick);
 
-		Project defaultDroneProject = new Project(context, projectName, landscapeMode);
-		File sceneDir = defaultDroneProject.getDefaultScene().getDirectory();
-
-		defaultDroneProject.setDeviceData(context); // density anywhere here
-		XstreamSerializer.getInstance().saveProject(defaultDroneProject);
-		ProjectManager.getInstance().setProject(defaultDroneProject);
-
 		backgroundImageScaleFactor = ImageEditing.calculateScaleFactorToScreenSize(
 				R.drawable.drone_project_background, context);
 
-		File backgroundFile = ResourceImporter.createImageFileFromResourcesInDirectory(context.getResources(), R.drawable.ic_video, sceneDir,
+		File backgroundFile = ResourceImporter.createImageFileFromResourcesInDirectory(context.getResources(), R
+						.drawable.ic_video, imageDir,
```

Comment:
please fix all this formatting issues your Android studio settings caused in this file while editing this lines.

Reply:
how?

Reply:
my android studio java formatting settings are fine. I don't know what case this!